tests/testthat/test-derive_locf_records.R

```diff
@@ -0,0 +1,268 @@
+## Test 1: new locf observations are derived correctly when visits are missing ----
+test_that("derive_locf_records Test 1: new locf observations are derived correctly when
+          visits are missing", {
```

Comment:
Please do not use line breaks in the test description. Otherwise, calling the "Format test_that test file" no longer works.
I think you can shorten the description by removing "new locf observations are derived correctly when".